napari/layers/base/base.py

```diff
@@ -369,7 +375,8 @@ def __init__(
             thumbnail=Event,
             status=Event,
             help=Event,
-            interactive=Event,
+            interactive=Event,  # Deprecated since 0.5.0
```

Comment:
I just found that it is not documented but for the deprecated event the `WarningEmitter` should be used. You could find example few lines bellow where the `select` event is defined.